metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableBackend.java

```diff
@@ -145,12 +141,8 @@ public BigtableBackend(
         final Groups groups,
         @Named("table") final String table,
         @Named("configure") final boolean configure,
-        @Named("maxWriteBatchSize") final int maxWriteBatchSize,
-        @Named("mutateRpcTimeoutMs") final int mutateRpcTimeoutMs,
-        @Named("readRowsRpcTimeoutMs") final int readRowsRpcTimeoutMs,
-        @Named("shortRpcTimeoutMs") final int shortRpcTimeoutMs,
-        @Named("maxScanTimeoutRetries") final int maxScanTimeoutRetries,
-        @Named("maxElapsedBackoffMs") final int maxElapsedBackoffMs,
+        @Named("metricsConnectionSettings")
```

Comment:
Hm, maybe I'm missing something but I don't think you need to use @Named here if you have only one **MetricsConnectionSettings** to be provided.

Reply:
Probably right. I will investigate.

Reply:
I slightly prefer `@Named` since it makes the dagger injection more explicit, but yeah it shouldn't be needed.